### ISSUE TITLE
PySide2 performance problem (not using commercial wheels is slower)

### ISSUE THREAD

**Issue by OverDriveGain** [Nuitka/Nuitka#1935]
**Python, nuitka, and OS info**
1.2.6
Commercial: None
Python: 3.8.10 (default, Nov 14 2022, 12:59:47) 
Flavor: Debian Python

**Installed via**
sudo apt-get install python3
pip3 install nuitka

**Issue description**
Django runs totally slow when compiled with nuitka. 

**Code example**
```
start = time.time()
out = []
rows = Model.objects.select_related().prefetch_related('items').all()
for row in rows:
    prefetch_item_list = getattr(row, 'items').all()
    for prefetch_item in prefetch_item_list:
        row._items.append(prefetch_item)
    out.append(row)
end = time.time()
```

I have around 2500 entries retrieved using the code above using ORM in **django**. Model is a **django.db.model** object. The same code takes around 1 seconds without being compiled, but it takes around 1 minutes when compiled with nuitka. 

Can you lead me into where I can fix/debug the problem?. I have really no clue how to even find a workaround, or what should I replace in my code.

**Comment by kayhayen** [comment]
Can you make the example a bit more complete, like populating a database first to demonstrate the issue. Such that I can see it myself. This sounds very bad indeed, but I would need to see it for myself. Maybe some form of paralellism that is done in Django normally, does not get applied for some reason.

**Comment by OverDriveGain** [comment]
Thank you so much for the response! 

I apologize. I tried to create a minimum reproducible code and during the process I figured that the problem arises not because of django, but rather by some bad practice of using signals in pyside2. Still, I am wondering how come in a native Python code the problem doesn't persist, but only when I compile it. Anyways, I can fix the problem now.

I am guessing that reproducing the issue won't be interesting for you anymore, since I read in your previous posts that pyside2 hasn't been a good citizen, and wont accept contributions. Also, it seems that I had bad code so its from my side.

Thank you also, for providing nuitka. Such a nice platform. I wish it solves issues I had with pyinstaller, such as the program being detected as a virus.
